pyanaconda/ui/common.py

```diff
@@ -391,6 +391,49 @@ def finished(self):
         """
         pass
 
+    # Initialization controller related code
+    #
+    # - initialization_controller
+    # -> The controller for this spokes and all others on the given hub.
+    # -> The controller has the init_done signal that can be used to trigger
+    #    actions that should happen once all spokes on the given Hub have
+    #    finished initialization.
+    # -> If there is no Hub (standalone spoke) the property is None
+    #
+    # - initialize_start()
+    # -> Should be called when Spoke initialization is started.
+    # -> Needs to be called explicitly, if we called it for every spoke by default
+    #    then any spoke that does not call initialize_done() would prevent the
+    #    controller form ever triggering the init_done signal.
+    #
+    # - initialize_start()
+    # -> Must be called by every spoke that calls initialize_start() or else the init_done
+    #    signal will never be emitted.
+
+    @property
+    def initialization_controller(self):
+        # standalone spokes don't have a category
+        if self.category:
+            return controller.get_controller(category_name=self.category.__name__)
```

Comment:
The call of `controller.get_controller()` is a little confusing. Same for `controller.controllers.get()`. Btw, I cannot find an import command for `controller` :-)